packages/react/src/experimental/Await.spec.tsx

```diff
@@ -96,3 +96,30 @@ describe('useAwait', () => {
     await waitFor(() => expect(screen.queryByText(TEXT)).toBeInTheDocument())
   })
 })
+
+describe('<Await>', () => {
```

Comment:
I understood @okinawaa's comment as using with /
```suggestion
describe('<Await/>', () => {
```

Reply:
[1cdfd91](https://github.com/suspensive/react/pull/169/commits/1cdfd9186b20338de2c8c07153df62192d18a099)